lib/src/textarea/types.ts

```diff
@@ -68,9 +68,12 @@ type Props = {
    */
   onBlur?: (val: { value: string; error?: string }) => void;
   /**
-   * If it is defined, the component will change its appearance, showing
-   * the error below the textarea. If it is not defined, the error
-   * messages will be managed internally, but never displayed on its own.
+   * If it is a defined value and also a truthy string, the component will 
+   * change its appearance, showing the error below the textarea. If the 
+   * defined value is an empty string, it will reserve a space below the
+   * component for a future error, but it would not change its look. In 
+   * case of being undefined or null, both the appearance and the space for 
+   * the error message would not be modified.
    */
   error?: string;
```

Comment:
Remember to also update here `pattern`, `minLength` and `maxLength` prop type descriptions. In the case of the Textarea, we also need to update the documentation of the component... 😢